pkg/devfile/parser/data/1.0.0/components.go

```diff
@@ -204,6 +204,15 @@ func convertV1ProjectToCommon(p Project) common.DevfileProject {
 
 }
 
+//GetStarterProjects returns the DevfileStarterProject parsed from devfile
+func (d *Devfile100) GetStarterProjects() []common.DevfileStarterProject {
+	return nil
+}
+
+func (d *Devfile100) AddStarterProjects(projects []common.DevfileStarterProject) error { return nil }
```

Comment:
Missing comments

Reply:
it is devfile V1 package, we are just adding functions to satisfy interface, We are not using them and eventually be removed.